modules/parse_poses.py

```diff
@@ -33,11 +33,11 @@ def get_root_relative_poses(inference_results):
     num_kpt_panoptic = 19
     num_kpt = 18
     for pose_id in range(found_poses.shape[0]):
-        if found_poses[pose_id, 3] == -1:  # skip pose if does not found neck
+        if found_poses[pose_id, 2] == -1:  # skip pose if is not found neck
```

Comment:
Seems it should be `if found_poses[pose_id, 5] == -1:`
This condition is before keypoint repacking. So now the pose is skipped if does not find nose

Reply:
You are right! Thanks, updated.